zerver/lib/actions.py

```diff
@@ -3351,17 +3351,17 @@ def do_change_logo_source(realm: Realm, logo_source: str, night: bool) -> None:
                    dict(type='realm',
                         op='update_dict',
                         property="logo",
-                        data=dict(logo_source=realm.logo_source,
-                                  logo_url=realm_logo_url(realm, night))),
+                        data=dict(realm_logo_source=realm.logo_source,
+                                  realm_logo_url=realm_logo_url(realm, night))),
```

Comment:
Hmm... So apparently we can't do this! So you should find an alternative to this.